python/cudf_polars/cudf_polars/containers/column.py

```diff
@@ -63,6 +68,35 @@ def obj_scalar(self) -> plc.Scalar:
             )
         return plc.copying.get_element(self.obj, 0)
 
+    def rename(self, name: str, /) -> Self:
+        """
+        Return a shallow copy with a new name.
+
+        Parameters
+        ----------
+        name
+            New name
+
+        Returns
+        -------
+        Shallow copy of self with new name set.
+        """
+        new = self.copy()
+        new.name = name
+        return new
+
+    def unname(self) -> Self:
```

Comment:
Maybe `remove_name`? `unname` sounds made up :) Not a big deal to me either way though.